app/poetry.lock

```diff
@@ -1,12 +1,11 @@
-# This file is automatically @generated by Poetry 2.1.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.0 and should not be changed by hand.
```

Comment:
Should we be using the same Poetry version 2.x.x? I notice a lot of `groups = ["main"]` lines being removed, but I don't know why.